x/exchange/spec/03_messages.md

```
@@ -299,6 +300,26 @@
+A market can transfer funds from one market to another using the `MarketTransferCommitments` endpoint.
+The `admin` must have the `PERMISSION_CANCEL` permission in the market (or be the `authority`).
+
+It is expected to fail if:
+* The currentmarket does not exist.
+* The newmarket does not exist.
+* The `admin` does not have `PERMISSION_CANCEL` in the market, and is not the `authority`.
+* One or more of the amounts is more than what is currently committed by the associated `account`.
+
+#### MsgMarketTransferCommitmentsRequest
+
++++ https://github.com/provenance-io/provenance/blob/v1.20.0/proto/provenance/exchange/v1/tx.proto#L324-L344
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)